validate.sh

```diff
@@ -16,3 +16,8 @@ mvn -q dependency:copy -Dartifact=${GROUPID}:${ARTIFACTID}:${VERSION}:pom -Doutp
 mvn -q dependency:copy -Dartifact=${GROUPID}:${ARTIFACTID}:${VERSION} -DoutputDirectory=${TARGET}
 mvn -q dependency:copy-dependencies -f ${TARGET}/${ARTIFACTID}-${VERSION}.pom -DoutputDirectory=dependencies
 java -cp ${TARGET}/\*:${TARGET}/dependencies/\* ${MAIN} ${ghprbAuthorRepoGitUrl} ${ghprbSourceBranch}
+
+echo "Running https://github.com/rhoar-ci/booster-catalog-verifier.git"
+git clone https://github.com/rhoar-ci/booster-catalog-verifier.git
```

Comment:
You can add `--depth=1` to speed up the clone by not grabbing the whole history

Reply:
Good point, though the history is very small ATM.